config/templates/eventing/brokers.yaml

```diff
@@ -1,11 +1,12 @@
 # Copyright KubeArchive Authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+
 apiVersion: eventing.knative.dev/v1
 kind: Broker
 metadata:
-  name: {{ .Release.Name }}-broker
-  namespace: {{ .Release.Namespace }}
+  name: kubearchive-broker
+  namespace: kubearchive
```

Comment:
Should we have here the `app.kubernetes.io/part-of: kubearchive` label?

Reply:
Added